test/classification/training/loggers/test_log_manager.py

```diff
@@ -1,31 +1,26 @@
-import os
 import shutil
+import tempfile
 import unittest
 
-from snorkel.classification.training import Checkpointer, LogManager
-
-TEST_LOG_DIR = "test/classification/training/loggers/logs"
+from snorkel.classification.snorkel_classifier import SnorkelClassifier
+from snorkel.classification.training import Checkpointer, LogManager, LogWriter
 
 
 class TestLogManager(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        if not os.path.exists(TEST_LOG_DIR):
-            os.makedirs(TEST_LOG_DIR)
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
 
-    @classmethod
-    def tearDownClass(cls):
-        if os.path.exists(TEST_LOG_DIR):
-            shutil.rmtree(TEST_LOG_DIR)
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
 
     def test_log_manager_points(self) -> None:
         """Unit test of log_manager (points)"""
         log_manager_config = {"counter_unit": "points", "evaluation_freq": 10}
 
         checkpointer = Checkpointer(
-            checkpoint_dir=TEST_LOG_DIR, checkpoint_factor=2, **log_manager_config
+            checkpoint_dir=self.test_dir, checkpoint_factor=2, **log_manager_config
         )
-        log_manager = LogManager(
+        log_manager = LogManager(  # type: ignore
```

Comment:
Is it the kwargs that make this not type checkable?

Reply:
yes